pyrsa/rdm/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .rdms import RDMs
+from .rdms import rank_transform
```

Comment:
Maybe it's an idea to have a separate transforms module, for things like rank transforms and MDS